src/index.js

```diff
@@ -124,6 +124,12 @@ export default (apiUrl, userSettings = {}) => (type, resource, params) => {
       // Add the reference id to the filter params.
       query[`filter[${params.target}]`] = params.id;
 
+      // Add sort parameter
+      if (params.sort && params.sort.field) {
```

Comment:
Are we sure that there will always only be one single sorting parameter? Or does React Admin allow multiple parameters?

Reply:
Yep, right now React Admin allows only one sorting field